python/revng/model/_util.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+
+def enum_value_to_index(enum_value: Enum):
+    """Converts an enum value to its index"""
+    return list(enum_value.__class__.__members__).index(enum_value.value)
+
+
+def force_constructor_kwarg(base_class: type, kwarg_name, kwarg_value):
```

Comment:
```suggestion
# TODO: once tuple_tree_generator becomes kind-aware, this will no longer be necessary
def force_constructor_kwarg(base_class: type, kwarg_name, kwarg_value):
```